target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -434,6 +434,11 @@ define LegacyDevice/WPE72_16M
 endef
 LEGACY_DEVICES += WPE72_16M
 
+define LegacyDevice/WNR2000
```

Comment:
all the netgear routers are still on the old code. converting them is a different patch i guess